features.md

```diff
@@ -0,0 +1,60 @@
+# Features Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+A feature is the representation of a business goal used in the [Authorization endpoint](authorizations.md) to declare what an user can do in a specific scope.
+
+## Main Endpoint
+**/api/authz/features**
+
+List all the available features in the system. Access is restricted to system administrators.
+
+Parameters:
+* page, size [see pagination](README.md#Pagination)
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators can access
+
+## Single Feature
+**/api/authz/features/<:string>**
+
+Provide detailed information about a specific feature. Access is restricted to system administrators. The JSON response document is as follow
+
+```json
+{
+  "id": "withdrawnitem",
```

Comment:
Is this a typo or is the intention here to describe the state after executing the feature?
I would expect this to be the name of the action, e.g. `withdrawItem` instead of the state after it has been performed

Reply:
it was a linguistic bug :)